test/E2E/Azure.Functions.PowerShellWorker.E2E/Azure.Functions.PowerShellWorker.E2E/Helpers/CosmosDBHelpers.cs

```diff
@@ -16,69 +15,74 @@ public class TestDocument
 
     public static class CosmosDBHelpers
     {
-        private static DocumentClient _docDbClient;
-        private static Uri inputCollectionsUri = UriFactory.CreateDocumentCollectionUri(Constants.CosmosDB.DbName, Constants.CosmosDB.InputCollectionName);
-        private static Uri outputCollectionsUri = UriFactory.CreateDocumentCollectionUri(Constants.CosmosDB.DbName, Constants.CosmosDB.OutputCollectionName);
-        private static Uri leasesCollectionsUri = UriFactory.CreateDocumentCollectionUri(Constants.CosmosDB.DbName, Constants.CosmosDB.LeaseCollectionName);
+        private static CosmosClient _cosmosDbClient;
+
+        private class Document
+        {
+            public string id { get; set; }
+        }
 
         static CosmosDBHelpers()
         {
             var builder = new System.Data.Common.DbConnectionStringBuilder();
             builder.ConnectionString = Constants.CosmosDB.CosmosDBConnectionStringSetting;
-            var serviceUri = new Uri(builder["AccountEndpoint"].ToString());
-            _docDbClient = new DocumentClient(serviceUri, builder["AccountKey"].ToString());
+            var serviceUri = builder["AccountEndpoint"].ToString();
+            _cosmosDbClient = new CosmosClient(serviceUri, builder["AccountKey"].ToString());
         }
 
         // keep
         public async static Task CreateDocument(string docId)
         {
             Document documentToTest = new Document()
             {
-                Id = docId
+                id = docId
```

Comment:
Should we continue to use Pascal case? E.g. `Id`.

Reply:
Documents uploaded to Cosmos with the new API must have a field called "id" (lower-case). I could implement this with a custom serializer, but it would be a lot of lines of code for a single character of syntax

Reply:
Let's keep it as is. Please include a comment that the Cosmos API expects a field `id`.

Reply:
Added